niceman/interface/diff.py

```diff
@@ -81,9 +93,10 @@ def __call__(prov1, prov2):
     
             pkgs_only_1 = pkgs_1_s - pkgs_2_s
             pkgs_only_2 = pkgs_2_s - pkgs_1_s
-    
+
             if pkgs_only_1 or pkgs_only_2:
-                print(pkg_type + 's:')
+                print(_make_plural(pkg_type) + ':')
+
```

Comment:
When I run diff on a spec with Git repos, it fails because the code below assumes a `version` attribute.

Reply:
There were a number of tests that were failing; I've fixed these.